dat/spob/mord.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<spob name="Mord">
+ <lua>spob/lua/proteron_mil_restricted.lua</lua>
+ <pos x="3242.477650" y="4168.187199"/>
+ <GFX>
+  <space>A06.webp</space>
+  <exterior>citadel.webp</exterior>
+ </GFX>
+ <presence>
+  <faction>Proteron</faction>
+  <base>90.000000</base>
+  <bonus>60.000000</bonus>
+  <range>0</range>
+ </presence>
+ <general>
+  <class>M</class>
+  <population>1e+06</population>
+  <hide>1.000000</hide>
+  <services>
+   <land/>
+   <refuel/>
+   <missions/>
+   <outfits/>
+   <shipyard/>
+  </services>
+  <commodities/>
+  <description>Mord is the Proteron manufactory and testing centre for spaceships. You can see ships landing and taking off every few minutes from the many spaceports dotting this world.</description>
```

Comment:
```suggestion
  <description>Mord is the Proteron manufactory and testing center for spaceships. You can see ships landing and taking off every few minutes from the many spaceports dotting this world.</description>
```

Reply:
Again, I'm sorry if I didn't notice, but is there a particular English standard we are obligated to follow?